python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 # Set the keras backend if not set, default is theano
-if not "KERAS_BACKEND" in os.environ:
+if "KERAS_BACKEND" not in os.environ:
     os.environ["KERAS_BACKEND"] = "theano"
 
 from keras.layers.core import Dense, Reshape, Dropout
```

Comment:
@maskani-moh : Tensorflow seems to be harder to install (especially on macOS and Windows...) so I've decided to use Theano and to switch defaut backend (tensorflow) to theano. You need to do it before importing keras.